src/utils/emailLink.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import A from '../atoms/a';
+
+const propTypes = {
+    className: PropTypes.string,
+    email: PropTypes.string,
+    tabIndex: PropTypes.number,
+};
+
+const defaultProps = {
+    className: undefined,
+    email: undefined,
+    tabIndex: -1,
+};
+
+function EmailLink(props) {
+    const {
+        className,
+        email,
+        tabIndex,
+    } = props;
+
+    if (!email) {
+        return null;
+    }
+
+    const onClick = (event) => {
+        event.stopPropagation();
+
+        window.location.href = `mailto:${email}`;
+    };
+
+    return (
+        <A
+            className={className}
+            onClick={onClick}
+            tabIndex={tabIndex}
```

Comment:
See my note above about `id` prop.  `<A>` now supports an `id` prop (as of your merging of #146 ... thank you BTW ;-)

Reply:
Fixed.